.gitignore

```diff
@@ -0,0 +1,18 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Environments
+.env
+backend/.venv
+backend/.env
+backend/.env.local
+
+# mac
+.DS_Store
+**/.DS_Store
+
+# PyCharm project settings
+.idea/
+.idea
```

Comment:
Please add extra line to the end of the file

You can add rule to the pre-commit hooks
![image](https://github.com/dmitryzhurkovsky/theater-app/assets/61659047/ad2f1aeb-830b-44e9-b376-42a3d559b3aa)

Reply:
has been fixed